README.md

```diff
@@ -11,6 +11,20 @@ A Visual Studio Code extension that listens to twitch chat and allows viewers to
 
 ![](./resources/screenshot-example.gif)
 
+<a name="quickstart"></a>
+## Quickstart Guide
+
+
+Install [VS Code](https://code.visualstudio.com/Download) if you don't already have it.
+
+Install [Twitch Themer](https://marketplace.visualstudio.com/items?itemName=MichaelJolley.vscode-twitch-themer) from the Visual Studio Marketplace
+Connect Twitch Themer to Twitch from the [Command Palette](#command-palette) (Ctrl + Shift + P)
```

Comment:
Need to add an additional return here or it will render as one sentence.

Reply:
Added